supertokens_python/recipe/thirdparty/provider.py

```diff
@@ -173,6 +173,12 @@ def __init__(
                 Awaitable[None],
             ]
         ] = None,
+        validate_access_token: Optional[
```

Comment:
adding it here causes a change in the positional arguments, since `generate_fake_email` gets pushed by one position.

These classes are a part of the public interface, therefore this becomes a breaking change! Please think about these next time.